### PR TITLE
[TOPI] Intel graphics conv2d autotvm template added

### PR DESCRIPTION
**Pull request by Laurawly**
@kevinthesun @yzhliu Could you review?

### CODE REVIEW

File: topi/python/topi/intel_graphics/conv2d.py [hunk 1]
```diff
@@ -53,40 +154,76 @@ def tile_and_bind3d(s, tensor, z, y, x, z_factor=2, y_factor=None, x_factor=None
...
+        _get_default_config(cfg, data, kernel, strides, padding, out_dtype, is_depthwise)
+
+    ic_bn, oc_bn = cfg["tile_ic"].size[-1], cfg["tile_oc"].size[-1]
+
+    new_attrs[layout_name] = 'NCHW%dc' % ic_bn
+    new_attrs['out_layout'] = 'NCHW%dc' % oc_bn
+
+    new_data = tvm.placeholder((batch_size, in_channel//ic_bn, height, width, ic_bn),
+                               dtype=data.dtype)
+    if is_depthwise:
```
Comment (kevinthesun) [inline topi/python/topi/intel_graphics/conv2d.py]: return None, and this should be moved up so that depthwise con2d doesn't query _get_default_config.

**Comment by Laurawly** [comment]
@tqchen What's the cause of ci time out error? Shall I trigger CI again?

**Comment by Laurawly** [comment]
@kevinthesun @yzhliu ready for review

**Comment by kevinthesun** [comment]
Do we want to add any test cases?

**Comment by Laurawly** [comment]
> Do we want to add any test cases?

I don't think CI has intel graphics backend.

**Comment by kevinthesun** [comment]
OK. For now let's just test locally.

**Comment by Laurawly** [comment]
> OK. For now let's just test locally.

Sure, local tests passed.

**Comment by kevinthesun** [comment]
Thanks!

**Pull request merged**